backend/routes/flightStatus.js

```diff
@@ -0,0 +1,44 @@
+import { API_KEY, API_SECRET } from "../config.js";
+import Amadeus from "amadeus";
+import express from "express";
+import { PrismaClient } from '@prisma/client'; 
+
+const prisma = new PrismaClient();
+const router = express.Router();
+const amadeus = new Amadeus({
+    clientId: API_KEY,
+    clientSecret: API_SECRET,
+});
+
+//flight search for adding trip
+router.get('/flight-status', async (req, res) => {
+    try {
+        const { carrierCode, flightNumber, scheduledDepartureDate } = req.query;
+        if (!carrierCode || !flightNumber || !scheduledDepartureDate) {
+            return res.status(400).json({ error: 'Missing required parameters' });
+        }
+        const response = await amadeus.schedule.flights.get({
```

Comment:
What happens if the flight doesn't exist? Would it throw an exception here or just return empty resuts?